packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -271,7 +271,11 @@ export type ReqTypes = {
   getBlockHeaders: {query: {slot?: number; parent_root?: string}};
   getBlockRoot: BlockIdOnlyReq;
   publishBlock: {body: unknown};
-  publishBlockV2: {body: unknown; query: {broadcast_validation?: string}};
+  publishBlockV2: {
+    body: unknown;
+    query: {broadcast_validation?: string};
+    headers: {"eth-consensus-version"?: ForkName};
```

Comment:
you can't be sure this is a valid ForkName here

```suggestion
    headers: {"eth-consensus-version"?: string};
```